pkg/controller/disruption/disruption.go

```diff
@@ -689,7 +689,6 @@ func (dc *DisruptionController) getExpectedScale(pdb *policy.PodDisruptionBudget
 }
 
 func countHealthyPods(pods []*v1.Pod, disruptedPods map[string]metav1.Time, currentTime time.Time) (currentHealthy int32) {
-Pod:
```

Comment:
@gavinfish  you can ask some people to  review this pr on slack? such as https://kubernetes.slack.com/archives/CGXR5GX6Z

Reply:
sure